tests/TestHelpers/HttpRequestHelper.php

```diff
@@ -216,17 +216,17 @@ public static function sendRequest(
 				$client
 			);
 
-			if ($response->getStatusCode() >= 400 && $response->getStatusCode() !== self::HTTP_TOO_EARLY) {
+			if ($response->getStatusCode() >= 400 && $response->getStatusCode() !== self::HTTP_TOO_EARLY && $response->getStatusCode() !== self::HTTP_CONFLICT) {
 				$sendExceptionHappened = true;
 			}
 
 			if ($debugResponses) {
 				self::debugResponse($response);
 			}
 			$sendCount = $sendCount + 1;
-			$loopAgain = !$sendExceptionHappened && ($response->getStatusCode() === self::HTTP_TOO_EARLY) && ($sendCount <= $sendRetryLimit);
+			$loopAgain = !$sendExceptionHappened && ($response->getStatusCode() === self::HTTP_TOO_EARLY || $response->getStatusCode() === self::HTTP_CONFLICT) && $sendCount <= $sendRetryLimit;
 			if ($loopAgain) {
-				// we need to repeat the send request, because we got HTTP_TOO_EARLY
+				// we need to repeat the send request, because we got HTTP_TOO_EARLY or HTTP_CONFLICT
 				// wait 1 second before sending again, to give the server some time
 				// to finish whatever post-processing it might be doing.
 				\sleep(1);
```

Comment:
since we are getting `409` in CI only giving the response and message might be helpful for debugging and understanding the actual behaviour
```suggestion
				// to finish whatever post-processing it might be doing.
				if(409){
				        give response
				}
				\sleep(1);
```

Reply:
Yes, a message will be useful.
It should just be a "warning" or "information" message - the code will keep running and try again.
Normally nobody will see the message anyway, because the test scenario will usually still pass (the retry of the request will probably pass). So nobody will know to look in the drone output. But I don't know how to communicate this sort of information back to the PR - we can't post GitHub comments.